### ISSUE TITLE
'Reselect' cannot be resolved

### ISSUE THREAD

**Issue by pmadruga** [ryandrewjohnson/react-localize-redux#10]
```
ERROR in ./~/react-localize-redux/lib/modules/locale.js
Module not found: Error: Can't resolve 'reselect' in '[...]/node_modules/react-localize-redux/lib/modules'
```
I have temporarily solved it by installing 'reselect' manually.

Nevertheless, thanks for this library.

**Comment by ryandrewjohnson** [comment]
`reselect` is listed as a peer dependency, which as of npm v3+ you have to install yourself if your project doesn't already include them. If you're missing any peer dependencies you should see a message like this when you `npm install`.

```
├── UNMET PEER DEPENDENCY react@^15.0.0
├── UNMET PEER DEPENDENCY react-redux@^4.3.0 || ^5.0.0
├── UNMET PEER DEPENDENCY redux@^3.0.0
└── UNMET PEER DEPENDENCY reselect@^3.0.0
```

**Comment by pmadruga** [comment]
In that case, shouldn't it be in your README?

**Comment by ryandrewjohnson** [comment]
Thanks for the input. I have decided to move `reselect` to dependencies as it may not be as common in the typical react/redux project. I have just published a new version v2.2.1 that has this change.
